packages/nocodb/src/lib/db/sql-data-mapper/lib/sql/BaseModelSqlv2.ts

```diff
@@ -313,8 +314,10 @@ class BaseModelSqlv2 {
     qb.count(this.model.primaryKey?.column_name || '*', {
       as: 'count'
     }).first();
-
-    return ((await qb) as any).count;
+    const res = (await this.dbDriver.raw(
+      qb.toQuery().replaceAll('\\?', '?')
```

Comment:
Is `.replaceAll('\\?', '?')` required here ?  This can leads to problem if table name / alias contain `?` in pg database.

Reply:
Missed the sanitise part for column name. Also use `unsanitise` on the final query. Tested in all dbs.